pkg/prometheus/statefulset.go

```diff
@@ -47,6 +47,7 @@ const (
 	configFilename                  = "prometheus.yaml.gz"
 	configEnvsubstFilename          = "prometheus.env.yaml"
 	sSetInputHashName               = "prometheus-operator-input-hash"
+	googleCredsEnvar                = "GOOGLE_APPLICATION_CREDENTIALS"
```

Comment:
Let's simply call it this:
```suggestion
	googleCredentials                = "GOOGLE_APPLICATION_CREDENTIALS"
```